src/layers/pool.jl

```diff
@@ -144,6 +144,47 @@ end
 
 topk_index(y::Adjoint, k::Int) = topk_index(y', k)
 
+"""
+    WeigthAndSumPool(in_feats)
+
+WeigthAndSum sum pooling layer.
+Takes a graph and the node features as inputs, computes the weights for each node and perform a weighted sum.
```

Comment:
```suggestion
In the forward pass, takes a graph and the node features as inputs, computes the weights for each node and performs a weighted sum.
```
Should also describe how the weights are computed and describe the `in_feats` constructor argument.